do_get instead of do_GET in local http server

Should we not expect a do_GET function instead of a do_get (case difference)

https://github.com/googleapis/managedkafka/blob/b4e9773b0c29dd34ad5b36cb5c57574e3c05a2fe/kafka-auth-local-server/kafka_gcp_credentials_server.py#L96

While running the local python authentication server as implemented, we cannot get the OAUTH BEARER Token from the server by using the upstream Kafka Java client (as described in the Kafka for BigQuery documentation https://cloud.google.com/managed-kafka/docs/authentication-kafka). The POST (as well as GET actually) method is implemented by using lower case in the python defined functions. 

See below for the Java error stack. 

```
org.apache.kafka.common.security.oauthbearer.internals.secured.UnretryableException: java.io.IOException: The response code 501 and error response {<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
        "http://www.w3.org/TR/html4/strict.dtd">
<html>
    <head>
        <meta http-equiv="Content-Type" content="text/html;charset=utf-8">
        <title>Error response</title>
    </head>
    <body>
        <h1>Error response</h1>
        <p>Error code: 501</p>
        <p>Message: Unsupported method ('POST').</p>
        <p>Error code explanation: HTTPStatus.NOT_IMPLEMENTED - Server does not support this operation.</p>
    </body>
</html>
} was encountered reading the token endpoint response; will not attempt further retries
	at org.apache.kafka.common.security.oauthbearer.internals.secured.HttpAccessTokenRetriever.handleOutput(HttpAccessTokenRetriever.java:286)
	at org.apache.kafka.common.security.oauthbearer.internals.secured.HttpAccessTokenRetriever.post(HttpAccessTokenRetriever.java:194)
	at org.apache.kafka.common.security.oauthbearer.internals.secured.HttpAccessTokenRetriever.lambda$retrieve$0(HttpAccessTokenRetriever.java:169)
	at org.apache.kafka.common.security.oauthbearer.internals.secured.Retry.execute(Retry.java:70)
	at org.apache.kafka.common.security.oauthbearer.internals.secured.HttpAccessTokenRetriever.retrieve(HttpAccessTokenRetriever.java:160)
	... 27 more
Caused by: java.io.IOException: The response code 501 and error response {<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
        "http://www.w3.org/TR/html4/strict.dtd">
<html>
    <head>
        <meta http-equiv="Content-Type" content="text/html;charset=utf-8">
        <title>Error response</title>
    </head>
    <body>
        <h1>Error response</h1>
        <p>Error code: 501</p>
        <p>Message: Unsupported method ('POST').</p>
        <p>Error code explanation: HTTPStatus.NOT_IMPLEMENTED - Server does not support this operation.</p>
    </body>
</html>
```

--- a/kafka-auth-local-server/kafka_gcp_credentials_server.py
+++ b/kafka-auth-local-server/kafka_gcp_credentials_server.py
@@ -18,7 +18,6 @@ import base64
 import datetime
 import http.server
 import json
-
 import google.auth
 import google.auth.crypt
 import google.auth.jwt
@@ -26,7 +25,9 @@ import google.auth.transport.urllib3
 import urllib3
 
 
-_credentials, _project = google.auth.default()
+_credentials, _project = google.auth.default(
+    scopes=['https://www.googleapis.com/auth/cloud-platform']
+)
 _http_client = urllib3.PoolManager()
 
 
@@ -40,13 +41,11 @@ _HEADER = json.dumps(dict(typ='JWT', alg='GOOG_OAUTH2_TOKEN'))
 
 
 def get_jwt(creds):
-  print('Creds ' + creds.service_account_email)
   return json.dumps(
       dict(
           exp=creds.expiry.timestamp(),
           iss='Google',
           iat=datetime.datetime.now(datetime.timezone.utc).timestamp(),
-          scope='kafka',
           sub=creds.service_account_email,
       )
   )
@@ -61,8 +60,6 @@ def b64_encode(source):
 
 
 def get_kafka_access_token(creds):
-  print('token: ' + creds.token)
-  print('payload ' + b64_encode(get_jwt(creds)))
   return '.'.join(
       [b64_encode(_HEADER), b64_encode(get_jwt(creds)), b64_encode(creds.token)]
   )
@@ -71,7 +68,7 @@ def get_kafka_access_token(creds):
 def build_message():
   creds = valid_credentials()
   expiry_seconds = (
-      creds.expiry - datetime.now(datetime.timezone.utc)
+      creds.expiry - datetime.datetime.now()
   ).total_seconds()
   return json.dumps(
       dict(
@@ -90,14 +87,13 @@ class AuthHandler(http.server.BaseHTTPRequestHandler):
     self.send_header('Content-type', 'application/json')
     self.end_headers()
     message = build_message().encode('utf-8')
-    print(message)
     self.wfile.write(message)
 
-  def do_get(self):
+  def do_GET(self):
     """Handles GET requests."""
     self._handle()
 
-  def do_post(self):
+  def do_POST(self):
     """Handles POST requests."""
     self._handle()
 

